### PR TITLE
Tuned up a few odds and ends

### PR DESCRIPTION
**Pull request by maxwelljordanwhite**
- Removed inline HTML
- Fixed a few typos
- Tidied up MD formatting
- Added Pacmax to Other Lists
- Updated the header image
- Removed CONTRIBUTING.md & added the contents of that to the Contributing heading
- Added title
- Tried to make some words more consistent, like GitHub, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-<h1 align="center">
-	<img width="594" src="https://cdn.rawgit.com/derimagia/awesome-alfred-workflows/master/media/header.gif" alt="awesome-alfred-workflows">
-	<br>
-</h1>
+# Awesome Alfred Workflows
+
+![Awesome Alfred Workflows](preview.gif "Awesome Alfred Workflows")
```
Comment (derimagia) [inline README.md]: This isn't centered, although you may be able to remove the  tag for this. I personally like the old image better (without the wallpaper clipped in)

File: README.md [hunk 2]
```diff
@@ -95,41 +106,56 @@ A curated list of Awesome Alfred Workflows.
...
-- [Ctwise Alfred Workflows](https://github.com/ctwise/alfred-workflows) - Workflows from the Github user "ctwise".
+- [Ctwise Alfred Workflows](https://github.com/ctwise/alfred-workflows) - Workflows from the GitHub user "ctwise".
 - [Packal](http://www.packal.org/) - The biggest place to find Workflows.
-- [Vítor Galvão’s Alfred Workflows](https://github.com/vitorgalvao/alfred-workflows/) - Workflows from the Github user "vitorgalvao".
+- [Pacmax](https://pacmax.org) - Pacmax, a sort of Packal rewrite, is for Exploring & Sharing Great Alfred Workflows, Snippets & Themes.
+- [Vítor Galvão’s Alfred Workflows](https://github.com/vitorgalvao/alfred-workflows/) - Workflows from the GitHub user "vitorgalvao".
+
+### Contributing
+
+Please ensure your pull request adheres to the following guidelines:
```
Comment (derimagia) [inline README.md]: Can you just link to the contributing.md here instead of removing it? Most things on https://github.com/sindresorhus/awesome lists have it as a separate file

File: README.md [hunk 3]
```diff
@@ -10,83 +9,95 @@ A curated list of Awesome Alfred Workflows.
 [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
 ## Contents
+
 - [Communication](#communication)
 - [Developer](#developer)
-- [Helpers](#helpers)
+- [Text Manipulation](#text-manipulation)
```
Comment (derimagia) [inline README.md]: It was alphabetical before, reason for change?

File: README.md [hunk 4]
```diff
@@ -95,41 +106,56 @@ A curated list of Awesome Alfred Workflows.
...
+### Other Lists
+
+Here is a list of other collections of Alfred resources:
 
 - [AlfredWorkflow.com](http://alfredworkflow.com) - List of Alfred Workflows.
-- [Ctwise Alfred Workflows](https://github.com/ctwise/alfred-workflows) - Workflows from the Github user "ctwise".
+- [Ctwise Alfred Workflows](https://github.com/ctwise/alfred-workflows) - Workflows from the GitHub user "ctwise".
 - [Packal](http://www.packal.org/) - The biggest place to find Workflows.
-- [Vítor Galvão’s Alfred Workflows](https://github.com/vitorgalvao/alfred-workflows/) - Workflows from the Github user "vitorgalvao".
+- [Pacmax](https://pacmax.org) - Pacmax, a sort of Packal rewrite, is for Exploring & Sharing Great Alfred Workflows, Snippets & Themes.
```
Comment (derimagia) [inline README.md]: I'm completely fine with adding this on here, can you move it to a separate PR?

**Comment by deanishe** [comment]
At this time I would advise against adding Pacmax.org to the list.

It has the right idea of just submitting the URL of a workflow's repo, but unfortunately, it doesn't handle them properly: All its download links on are to `<repo>/master/zip`, not to the proper download URLs for any workflow.

Some workflows do include an `.alfredworkflow` file in the repo, but the majority (in my experience) use GitHub as intended and upload them under releases.

The upshot is that workflow authors are likely face a bunch of support requests from folks who clicked "download" on Pacmax.org and don't understand what to do next, and including Pacmax in this list can only exacerbate the problem.

File: README.md (hunk 4 above)
Comment (maxwelljordanwhite) [reply README.md]: Sure! I've removed this from my next commit and will create a new PR once I've better implemented the way Pacmax handles the way it works with GitHub

File: README.md (hunk 2 above)
Comment (maxwelljordanwhite) [reply README.md]: Sure, adding this back in my next commmit

File: README.md (hunk 3 above)
Comment (maxwelljordanwhite) [reply README.md]: Whoops! Not sure what happened there, my bad. Should have been organized alphabetically, until Other Lists. My next commit should correct this but let me know if you'd prefer it differently

File: README.md (hunk 1 above)
Comment (maxwelljordanwhite) [reply README.md]: I went ahead and reverted this so that it's centered and also tried to recreate the gif without the background (better cropped) in my next commit

File: README.md [hunk 5]
```diff
@@ -22,71 +22,81 @@ A curated list of Awesome Alfred Workflows.
...
-- [Pandoctor](https://github.com/smargh/alfred_pandoctor) - Alfred interface to [pandoc](http://pandoc.org), a tool that allows to convert between text formats (Markdown, LaTeX, DOC, DOCX, ...).
-- [Unicode Text Search](https://github.com/bevesce/unicode-symbols-search) - Search for Unicode Symbols.
+### Helpers
 
-## Scientific
-- [Convert](https://github.com/deanishe/alfred-convert) - Offline conversion of units and (crypto)currencies.
-- [DOITools](https://github.com/hbuschme/doi-tools-alfred-workflow/) - Tools to resolve, open, shorten DOI numbers and even convert them to bibtex.
-- [Scientific Workflow](https://github.com/andrewning/alfred-workflows-scientific) - Workflows that are useful for scientists: e.g. LaTeX, importing references in bibtex.
-- [Skim remote](http://www.packal.org/workflow/skim-remote) - Controls the [Skim PDF Viewer](http://skim-app.sourceforge.net) remotely from Alfred.
+- [Alfred-Workflow](https://github.com/deanishe/alfred-workflow) - Python Library for writing Alfred workflows.
```
Comment (derimagia) [inline README.md]: Can you swap helpers back? I know this one was an exception (or rename it to Workflow Helpers). These are more development links than user ones.

**Comment by derimagia** [comment]
@deanishe Thanks for pointing.  that our. I jumped a little bit too quickly at a packal replacement but you're right we should be careful. We will hold off it for now.

I saw this project and responded to it on [reddit](https://www.reddit.com/r/macapps/comments/asmvxs/fellow_alfred_lovers_ive_been_working_on_a_sort/egvycfs/). 

Another future suggestion may be to see if you can link to virus total if it's easy (https://www.virustotal.com) but since it's manual process you can probably hold off on it. Good luck again

**Comment by deanishe** [comment]
What's the reason for changing all the category headers to level 3 (###)?

By implication of the semantics of HTML, that makes them sub-sections of the "Contents" section (level 2).
